docs/03-extensions/theme-chocolatine/reference/css-variables.mdx

```diff
@@ -1,15 +1,98 @@
 ---
 title: "CSS variables"
 description:
-  "This reference lists all the CSS variables useful for customizing the theme
-  Chocolatine."
+  "This reference lists all the CSS variables useful for customizing the theme."
 sidebar_position: 1
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+Front-Commerce's theme Chocolatine uses a set of CSS variables defined in
+`theme/_variables.scss` to ensure a certain consistency in the application's
+design. We recommend to reuse these variables as much as possible, and adapting
+them to your project's design choices. See
+[Override a stylesheet](/docs/remixed/guides/override-a-stylesheet) for more
+details about adapting your application to your brand.
 
-What variables are provided by Front-Commerce and what are their usages.
+# Colors
 
-:::info
+Defined in `theme/components/atoms/Colors/_colors.scss`, color variables are
+useful to ensure a consistent design in your app.
+
+## Base colors variables
+
+Those variables are often used to define other meaningful variables, and serve
+as a root color definition base for your theme. They are used in a wide variety
+of situation in Theme Chocolatine, from text color to backgrounds.
+
+- `white`, `black` and `red`
```

Comment:
**nitpick:** I think it'd be clearer if CSS variables had the `--` prefix. It'll be closer to what developers view in their code.

**suggestion:** 
```suggestion
- `--white`, `--black` and `--red`
```